db/migrate/20241101152410_drop_benefits_intake_uuid_index_from_form_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class DropBenefitsIntakeUuidIndexFromFormSubmissions < ActiveRecord::Migration[7.1]
+  def change
```

Comment:
Please add   disable_ddl_transaction!

Reply:
Ok added, thanks! Do I need to re-run the migrations?

Reply:
no, you don't.